### DISCUSSION TITLE
Set default address country based on store location

### DISCUSSION THREAD

**Discussion by MatthiasBrodelet** [craftcms/commerce#3618] [Ideas] [answered]
When creating a manual order in the CP and adding a new address the United states is always the default country option.
Our client will never use this and Belgium (in our case) as a default will be a lot better.

It would be nice if this was based on the store location country.
It's not a big issue, but it's annoying to always change this to another country when 90% of the orders will be in Belgium.

<img width="1441" alt="Screenshot 2024-08-08 at 16 16 45" src="https://github.com/user-attachments/assets/8398c24f-d444-4677-a9ce-e805a95f043b">

**Comment by nfourtythree** [comment, marked as answer]
Hi @MatthiasBrodelet 

This feature was added in Commerce version `5.2.0`

Thanks!
